test/test_sparse_csr.py

```diff
@@ -2004,33 +2032,102 @@ def test_compressed_layout_conversions_coverage(self, device, from_layout, to_la
 
+    @skipMeta
+    @all_sparse_compressed_layouts()
+    def test_dense_to_from_sparse_compressed(self, device, layout):
+        """
+        This test tests conversion from dense to/from CSR and CSC
+        by comparing to SciPy's implementation.
+        """
+        if layout is torch.sparse_bsc:
+            # TODO: Remove this once support has been enabled
+            return
+        if layout is torch.sparse_bsr:
+            # TODO: Remove this once support has been enabled
+            return
+
+        for shape in [(0, 10), (6, 0), (6, 10), (0, 0)]:
+            dense = make_tensor(shape, dtype=torch.float, device=device)
+            dense = dense.relu()  # Introduce some sparsity
+            sp_matrix = self._construct_sp_matrix(dense, layout)
+            pt_matrix = self._convert_to_layout(dense, layout)
+
+            compressed_indices_mth = {
+                torch.sparse_csr: torch.Tensor.crow_indices,
+                torch.sparse_csc: torch.Tensor.ccol_indices,
+            }[layout]
+
+            plain_indices_mth = {
+                torch.sparse_csr: torch.Tensor.col_indices,
+                torch.sparse_csc: torch.Tensor.row_indices,
+            }[layout]
+
+            self.assertEqual(layout, pt_matrix.layout)
+            self.assertEqual(sp_matrix.shape, pt_matrix.shape)
+            self.assertEqual(torch.tensor(sp_matrix.indptr, dtype=torch.int64), compressed_indices_mth(pt_matrix))
+            self.assertEqual(torch.tensor(sp_matrix.indices, dtype=torch.int64), plain_indices_mth(pt_matrix))
+            self.assertEqual(torch.tensor(sp_matrix.data), pt_matrix.values())
+
+            self.assertEqual(dense, pt_matrix.to_dense())
+
+    @skipMeta
+    @all_sparse_compressed_layouts()
+    @coalescedonoff
+    @dtypes(torch.double)
+    def test_sparse_to_sparse_compressed(self, device, dtype, coalesced, layout):
+        """
+        This test tests conversion from COO to CSR and CSC
+        by comparing to SciPy's implementation.
+        """
+        if layout is torch.sparse_bsc:
+            # TODO: Remove this once support has been enabled
+            return
+        if layout is torch.sparse_bsr:
+            # TODO: Remove this once support has been enabled
+            return
+
+        for shape in [(0, 10), (6, 0), (6, 10), (0, 0)]:
+            sparse_dim = 2
+            nnz = shape[0] * shape[1] // 2
+            sparse, _, _ = self.genSparseTensor(shape, sparse_dim, nnz, coalesced, device, dtype)
+            sp_matrix = self._construct_sp_matrix(sparse, layout)
+            pt_matrix = self._convert_to_layout(sparse, layout)
+
+            compressed_indices_mth = {
+                torch.sparse_csr: torch.Tensor.crow_indices,
+                torch.sparse_csc: torch.Tensor.ccol_indices,
+            }[layout]
+
+            plain_indices_mth = {
+                torch.sparse_csr: torch.Tensor.col_indices,
+                torch.sparse_csc: torch.Tensor.row_indices,
+            }[layout]
```

Comment:
We use this method in many tests.
 
After https://github.com/pytorch/pytorch/pull/77530 lands, this code block can be replaced with
```suggestion
compressed_indices_mth, plain_indices_mth = sparse_compressed_indices_methods[layout]
```
No action is required here, just FYI.